com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -35,6 +35,10 @@ public override VisualElement CreateInspectorGUI()
             var volumeProp = serializedObject.FindProperty(() => Target.BoundingShape2D);
             ux.Add(new PropertyField(volumeProp));
             ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.Damping)));
+            var confiner = Target; // so it gets captured in the lambdas
+            if (confiner.IsCameraOversizedForTheConfiner())
```

Comment:
This doesn't work.  With UITK, you have to check IsCameraOversizedForTheConfiner in an event callback, not at UI creation time.

Reply:
That's why it did not work properly. Thanks